### ISSUE TITLE
Can't conntect to calls on v0.16.0

### ISSUE THREAD

**Issue by eldr0n** [mattermost/mattermost-plugin-calls#432]
It's not possibile to connetct to open calls. Tested on self-hosted 7.10/11.

```json
{
  "caller": "app/plugin_api.go:976",
  "level": "error",
  "msg": "callback failed: not found",
  "origin": "main.(*Plugin).handleLeave websocket.go:454",
  "plugin_id": "com.mattermost.calls",
  "timestamp": "2023-05-15 07:37:05.804 Z"
}
```
This was the only log related to calls.

**Comment by eldr0n** [comment]
@streamer45 I updated the plugin to 0.16 from 7.10, then it stopped working, I then updated mattermost to 7.11, which didn't help. Now I donwgradet calls to 0.15.1 and it works again.

**Comment by streamer45** [comment]
Thanks for confirming. On v0.16, was it timing out connecting or was it failing immediately?

**Comment by eldr0n** [comment]
It could not connect, the loading spinner in the widget was spinning endlessly.

**Comment by streamer45** [comment]
Okay, I wonder if there could be some network misconfiguration at play here. It's possible that explicitly setting the `RTC Server Address` could help. By the way, how are you deploying your MM server? Is it running in a container or else?

**Comment by eldr0n** [comment]
It is deployed with docker but I use my own nginx proxy. The RTC Server Address is set to 0.0.0.0.

**Comment by streamer45** [comment]
Thanks that's good info. I wonder if setting to `0.0.0.0` is what's causing issues. In the new plugin version the calls RTC service should listen to all available interfaces if the *RTC Server Address* setting is left empty. Although `0.0.0.0` is commonly used to achieve the same it may now behave differently. It may be worth a try to leave it blank, if you get a chance to upgrade again.

**Comment by eldr0n** [comment]
Ok, i will give it a try.

**Comment by eldr0n** [comment]
Nice, that actually worked.
Thank you for your support.

**Comment by streamer45** [comment]
Nice, thanks for letting us know.

@cpoile Could you create a ticket to track a fix for this? We should likely treat `0.0.0.0` as a special case to avoid more users from hitting the same issue.

**Comment by cpoile** [comment]
@streamer45 Done, thanks!
